### PR TITLE
Pal 1092 generate coderelease and tag

### PR DESCRIPTION
**Pull request by dev930018**

### CODE REVIEW

File: charts/attribute-masking-service/templates/_helpers.tpl [hunk 1]
```diff
@@ -102,7 +102,7 @@ Calculate a storage path based on the code release artifact id or the supplied v
 Calculate a storage name based on the code release artifact id or the supplied value of codeRelease
```
Comment (m78233) [inline charts/attribute-masking-service/templates/_helpers.tpl]: Need to replace the text "codeRelease" with a suitable alternative.  Will need to be done in each of the _helpers.tpl files

Comment (m78233) [inline charts/attribute-masking-service/templates/_helpers.tpl]: Found three lingering references in the code to codeRelease: /templates/_helpers.tpl;  create-kafka-queues/src/main/resources/values.yaml; values.yaml.

**Review by m78233 — CHANGES_REQUESTED** [review]
A couple of minor things to check-up on.

**Review by m78233 — DISMISSED** [review]

File: NOTICES.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ List of third-party dependencies grouped by their license type
 * spring-boot-starter-web ([org.springframework.boot:spring-boot-starter-web:2.3.1.RELEASE](https://spring.io/projects/spring-boot))
 * Spring Cloud Starter OpenFeign ([org.springframework.cloud:spring-cloud-starter-openfeign:2.2.2.RELEASE](https://projects.spring.io/spring-cloud))
 * event-logging ([uk.gov.gchq.eventlogging:event-logging:4.0.8_schema-v3.2.4](https://github.com/gchq/event-logging))
-* GCHQ Palisade Common Library ([uk.gov.gchq.palisade:common:0.5.0-RELEASE](https://github.com/gchq/Palisade-common))
+* GCHQ Palisade Common Library ([uk.gov.gchq.palisade:common:0.5.0-SNAPSHOT](https://github.com/gchq/Palisade-common))
```
Comment (developer01189998819991197253) [inline NOTICES.md]: shouldnt this be 0.5.1?

File: audit-service/mvn_dependency_tree.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
-uk.gov.gchq.palisade:audit-service:jar:0.5.0-RELEASE
-+- uk.gov.gchq.palisade:common:jar:0.5.0-RELEASE:compile
+uk.gov.gchq.palisade:audit-service:jar:0.5.0-SNAPSHOT
```
Comment (developer01189998819991197253) [inline audit-service/mvn_dependency_tree.txt]: needs to be rebuilt against 0.5.1 dependencies

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]

**Review by ac74475 — CHANGES_REQUESTED** [review]
doesnt work and negates the work that @m78233 did. Test it with the release profile and look at the tag on the docker images created

**Review by ac74475 — DISMISSED** [review]
it now works for the release profiles

**Comment by dev930018** [comment]
Changes to resource-service to fix sonarqube smell [here](https://palisade-prod.apps.gss.gov.uk/sonarqube/project/issues?id=Palisade-Services-PAL-1092-generate-coderelease-and-tag&issues=AXrixArNRy8TgjbXaE3v&open=AXrixArNRy8TgjbXaE3v)

File: licenses/mit_license.html [hunk 4]
```diff
@@ -18,7 +18,7 @@
 <link type="text/css" rel="stylesheet" href="https://opensource.org/files/css/css_7rptTjxTlnnwcQOT_3B6dmWJfzo0pnNKsCvuv3HArrk.css" media="all" />
 
 <!--[if (lt IE 9)]>
-<link type="text/css" rel="stylesheet" href="https://opensource.org/sites/all/themes/bootstrap-business/css/ie8.css?qqdhfs" media="all" />
```
Comment (developer01189998819991197253) [inline licenses/mit_license.html]: revert license change

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]
Approved pending license revert

File: services-manager/pom.xml [hunk 5]
```diff
@@ -142,7 +142,7 @@
                                 <replacement>
                                     <token>image.tag</token>
                                     <!--suppress UnresolvedMavenProperty -->
```
Comment (ac74475) [inline services-manager/pom.xml:144]: the readers.app.revision starts with a dash that needs removing to fix the JVM deployment

**Review by ac74475 — DISMISSED** [review]

**Pull request merged**